L1Trigger/L1TGlobal/plugins/L1TGlobalProducer.cc

```diff
@@ -139,6 +147,8 @@ L1TGlobalProducer::L1TGlobalProducer(const edm::ParameterSet& parSet)
   m_muInputToken = consumes<BXVector<Muon>>(m_muInputTag);
   if (m_useMuonShowers)
     m_muShowerInputToken = consumes<BXVector<MuonShower>>(m_muShowerInputTag);
+  if (m_saveAxoScore)
+    m_axoInputToken = consumes<BXVector<AXOL1TLScore>>(m_axoInputTag);
```

Comment:
My understanding is that it is needed for the ntuple config here? https://github.com/cms-sw/cmssw/blob/6a131029e88625000a80f6618e4a173d325fffc0/L1Trigger/L1TNtuples/python/L1NtupleEMU_cff.py#L55

Reply:
Sorry, this is indeed confusing! Is the L1tGlobalProducer producing the AXO score object, or reading it in?
For the former case, you do call the produces method, but only for the latter you need an InputTag and InputToken. Could you please clarify?

Reply:
> My understanding is that it is needed for the ntuple config here?
> 
> https://github.com/cms-sw/cmssw/blob/6a131029e88625000a80f6618e4a173d325fffc0/L1Trigger/L1TNtuples/python/L1NtupleEMU_cff.py#L55

I don't understanding this explanation, because `l1AXOEmuTree` is an instance of `L1AXOTreeProducer`, and here instead we are commenting on the plugin `L1TGlobalProducer`.

Reply:
I looked into this: the GlobalProducer indeed produces the AXO score, so I don't actually need the input tag and token. Thanks for pointing this out.